src/clean_air/data/extract_metadata.py

```diff
@@ -89,7 +89,8 @@ def extract_metadata(
     for cube in cubes:
         bounding_polygon = _cube_to_polygon(cube)
         total_polygon_list.append(bounding_polygon[0])
-        spatial_extent = SpatialExtent(bounding_polygon)
+        # TODO: Ignoring the CRS for now, but SpatialExtent should use it
+        spatial_extent = SpatialExtent(bounding_polygon[0])
```

Comment:
ahh, the issue here isn't the CRS, but that you were passing a tuple of Polygon and CRS.

Consider this:
```python
polygon = Polygon(...)
crs = CRS(...)
bounding_polygon = (polygon, crs)

SpatialExtent(bounding_polygon)  # tuple is passed as a single argument to bbox parameter  
SpatialExtent(bbox=bounding_polygon) # equivalent to the above

# as opposed to...
SpatialExtent(polygon, crs)  # arguments passed separately
SpatialExtent(bbox=polygon, crs=crs)
SpatialExtent(bounding_polygon[0], bounding_polygon[1])  # arguments passed separately, but just accessed via the tuple using indices
SpatialExtent(*bounding_polygon)  # using "tuple unpacking" to split the tuple up. Equivalent to the version above
```

So I think you might be able to pass the CRS in after all! (Unless what you meant is you had tried passing in the CRS correctly as above and you encountered other issues)

Reply:
I think it's more that I had written it anticipating `SpatialExtent` would take a tuple, as passing both the bbox and crs seemed useful. But as it doesn't (yet) I was just passing in the bbox for now - does that make sense as a thing to do? Or is it better to change to one of the above?

Reply:
I think that whilst passing a tuple here is convenient, in general I suspect there are more cases where we'd end up creating a tuple just to pass in, which seems a bit redundant.

I'd go with something like this:
```python
        bounding_polygon, bounding_polygon_crs = _cube_to_polygon(cube)
        total_polygon_list.append(bounding_polygon)
        spatial_extent = SpatialExtent(bounding_polygon, bounding_polygon_crs)
```
Unpacking the tuple returned by `_cub_to_polygon` straight into meaningfully named variables helps the reader understand what _cube_to_polygon is returning, so it improves the readability of the code.